ui-components/dialog.md

```diff
@@ -15,48 +15,49 @@ A dialog is a TinyMCE UI component. Dialogs have their own dialog components whi
 
 * **Display complex information** - These dialogs can display complex information by using layouts components like tabs or columns to help present information to the user (e.g., help dialog or special characters dialog are tabbed dialogs).
 
-* **Interactive dialogs** - These dialogs use web forms to collect interaction data and then apply the data (e.g.: search and replace dialog, uses an input field. Where the input text will be used as the search key. Another example is, special characters or character map dialogs use typeaheads to dynamically narrow down matches as you type)
+* **Interactive dialogs** - These dialogs use web forms to collect interaction data and then apply the data (e.g.: search and replace dialog, uses an input field, where the input text will be used as the search key. Another example is, special characters or character map dialogs use typeaheads to dynamically narrow down matches as you type)
```

Comment:
We should probably make this "apply the data. For example, search and replace dialog...". Having so much in brackets seems odd.